### PR TITLE
[Range, Worksheet] (Format, AutoFilter) Adding samples for cell sizing, autofilters, and tab color

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR adds samples to the areas described in the title. The API selection was based off of a list of commonly used Excel calls.

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml [hunk 1]
```diff
@@ -6,7 +6,29 @@ fullName: ExcelScript.AutoFilter
...
+  /**
+   * This script creates an autofilter on the worksheet that filters out rows based on column values. 
+   * The autofilter filters to only include rows that have a value in column C in the lowest 10 values 
+   * (of column C values).
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    const currentSheet = workbook.getActiveWorksheet();
+    const dataRange = currentSheet.getUsedRange();
+
+    // Add a filter that will only show the rows with the top 10% of values in column C
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml:24]: This says top 10% of values, but the preceding description says lowest 10 values, and it's using bottomItems. Should this say lowest 10% of values? Or lowest 10 values?

File: docs/docs-ref-autogen/excel/excelscript/excelscript.rangeformat.yml [hunk 2]
```diff
@@ -262,7 +323,31 @@ methods:
...
+
+          ```TypeScript
+          /**
+           * This script doubles the row height for every row in the active worksheet's used range.
+           */
+          function main(workbook: ExcelScript.Workbook) {
+            const currentSheet = workbook.getActiveWorksheet();
+            const usedRange = currentSheet.getUsedRange();
+
+            // Get all the current row heights before setting them to improve performance.
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.rangeformat.yml:339]: To clarify the purpose of this loop, I recommend switching the order of the phrases: 
```suggestion
            // To optimize performance, get all the current row heights before setting them.
```

**Review by alison-mk — COMMENTED** [review]
I have one question that I think should be resolved before approving, and one minor suggestion.

File: docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml (hunk 1 above)
Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml:24]: The sample filters to the 10 lowest valued items. I'll update this comment accordingly.

**Comment by AlexJerabek** [comment]
Thanks @alison-mk! I've updated the samples based on your feedback and regenerated the docs.

**Review by alison-mk — APPROVED** [review]

**Pull request merged**